### ISSUE TITLE
Ability to detect currently viewed dash tab.

### ISSUE THREAD

**Issue by mrkbx** [Yellow-Dog-Man/Resonite-Issues#1012]
### Is your feature request related to a problem? Please describe.

I have been working on a custom dash indicator, and it kind of annoys me how there isn't really a way to detect what dash tab im currently viewing.

### Describe the solution you'd like

Flux node/component that outputs dash tabs that are currently focused on, allowing to detect and apply changes while viewing certain dash tabs (ex. making the indicator blue while browsing worlds)

### Describe alternatives you've considered

There have been alternatives, but they are not really possible to pull off since it involves the ability to install facets (and be able to see dash via userspace inspector) while in desktop mode.

**Comment by art0007i** [comment]
I personally would prefer for this feature to not exist as it would allow someone to make a tool that shows them what dash tab I'm currently looking at, which I don't want. The current way that requires installing a facet is ideal in my opinion as doing that is kind of a opt in and consent option.

**Comment by mrkbx** [comment]
> I personally would prefer for this feature to not exist as it would allow someone to make a tool that shows them what dash tab I'm currently looking at, which I don't want. The current way that requires installing a facet is ideal in my opinion as doing that is kind of a opt in and consent option.

That option doesn't work for desktop.

**Comment by Frooxius** [comment]
Are you looking for this node/component to work in world space? If that's the case, we would not add this, as this has privacy/security issues.

We could add such node/component and have it work specifically in userspace. If you really wanted to proxy the current tab, then you'd have to do it yourself with cloud variables.

**Comment by mrkbx** [comment]
>We could add such node/component and have it work specifically in userspace. If you really wanted to proxy the current tab, then you'd have to do it yourself with cloud variables.

A component seems better than having to do mental gymnastics :^/

**Comment by Chundelac** [comment]
I agree this should not be added for privacy reasons. There are currently community made tools to achieve something like this but you are right that they are not usable by desktop only users.
However that is another issue and I feel like making a specific workaround just for this use case is not the way. Allowing facets editing and usage for desktop users would be.
